roles/common/files/letsencrypt-gencert

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+service apache2 stop
```

Comment:
Moved to after for loop, good point.

I had debated about whether to start Apache again in this script. It is started by a later task in the same `letsencrypt.yml` tasks file. I hadn't re-started it here because I wasn't sure that in all cases (e.g on first run) it would be started yet here in the first place. But actually I think it will be (because it's an enabled service by default), and in any case there's no harm in starting it, and it's clearer to have a paired start/stop within this script. So I added a start at the end.